### ISSUE TITLE
[Help] Caddy use ingress network instead of custom network

### ISSUE THREAD

**Issue by mathisloge** [lucaslorentz/caddy-docker-proxy#49]
I´ve got some services which are setup with docker stack deploy. 
All of the necessary services are connected to a network called "proxy_network".

Caddy-docker recognize these services correctly but use the ip address of the ingress network instead of the proxy_network. Is there any way to block caddy-docker from using the ip addresses of the ingress network? 

Thank you in advance

**Comment by lucaslorentz** [comment]
Hey @mathisloge 

Definitely we should not consider the ingress network as an option for communication. I will fix it soon.

Do you also need to expose your services port directly (not behind caddy proxy)? If no, you can remove ports from your services and they will be removed from ingress network.

**Comment by mathisloge** [comment]
Sorry @lucaslorentz i have made a mistake. 
i´ve forgot to put the labels under the deploy tag. Now it runs smoothly.

**Comment by lucaslorentz** [comment]
Yes, that's another thing, that issue only happens when proxying to containers, instead of service VIP.

**Comment by mathisloge** [comment]
yeah. i´m still discorver docker and need to learn. Sorry!

**Comment by lucaslorentz** [comment]
Ha, I'm learning too. I just understood the ingress network, I didn't know it attaches to that network automatically when you expose a port.
